x-pack/plugins/cases/public/common/mock/kibana_react.mock.ts

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
What is the difference between this file and `x-pack/plugins/cases/public/common/lib/kibana/kibana_react.mock.ts` ?

Reply:
good eye, ill remove this one